modules/liveIntentIdSystem.js

```diff
@@ -136,17 +137,32 @@ export const liveIntentIdSubmodule = {
   decode(value, config) {
     const configParams = (config && config.params) || {};
     function composeIdObject(value) {
-      const base = { 'lipbid': value.unifiedId };
-      delete value.unifiedId;
-      return { 'lipb': { ...base, ...value } };
+      // old versions stored lipbid in unifiedId. Ensure that we can still read the data.
+      value.lipbid = value.nonId || value.unifiedId
```

Comment:
in case `nonId` is `0` the `unifiedId` value will be set
not sure if we could run into that case though
```suggestion
      value.lipbid = value.nonId ?? value.unifiedId
```

Reply:
This should not really matter, as idres responses should be well-formed.

Additionally, there is the check here https://github.com/LiveIntent/Prebid.js/pull/5/files#diff-489289275f4ee5264024b54c9c851e6c57684fd2d0bccab91d8b9ccef150e235R156